docs/docs/code/python/README.md

```diff
@@ -308,33 +311,38 @@ Sometimes you want to end your workflow early, or otherwise stop or cancel the e
 - You only want to run your workflow for users in the United States. If you receive a request from outside the U.S., you don't want the rest of the code in your workflow to run.
 - You may use the `user_id` contained in the event to look up information in an external API. If you can't find data in the API tied to that user, you don't want to proceed.
 
-**In any code step, calling `pd.flow.exit()` will end the execution of the workflow immediately.** No remaining code in that step, and no code or destination steps below, will run for the current event.
+**In any code step, calling `return pd.flow.exit()` will end the execution of the workflow immediately.** No remaining code in that step, and no code or destination steps below, will run for the current event.
+
+::: tip
+It's a good practice to use `return pd.flow.exit()` to immediately exit the workflow.
```

Comment:
@andrewjschuang this is a good tip!

Think we should add this to the Node.js `$.flow.delay` doc as well?